packages/ui/src/kit/VDateInput/VDateInput.vue

```diff
@@ -0,0 +1,228 @@
+<template>
+  <div ref="root" :class="vDateInputCssClass">
+    <div class="v-date-input-field">
+      <input-atom
+        v-bind="$attrs"
+        :value="formattedValue"
+        @focus="onFocusInput"
+        @blur="onBlurInput"
+      />
+      <div class="v-date-input-field-icon">
+        <v-svg-icon name="calendar" />
+      </div>
+    </div>
+    <div v-if="isCalendarVisible" class="v-date-input-calendar">
+      <section class="v-date-input-controls">
+        <button class="v-date-input-control" @click="onClickPreviousMonth">
+          <v-svg-icon name="chevron-left" />
+        </button>
+        <div class="v-date-input-now">
+          <span>
+            {{ formattedInnerMonth }}
+          </span>
+          <input v-model="innerYear" min="1940" type="number" />
```

Comment:
why min 1940?

Reply:
Which year in your opinion should be minimal?

Reply:
it's depends maybe from prop?
but not hardcoded in html node.